java/rocksjni/rocksjni.cc

```diff
@@ -1707,6 +1707,23 @@ void Java_org_rocksdb_RocksDB_dropColumnFamily(JNIEnv* env, jobject /*jdb*/,
   }
 }
 
+/*
+ * Class:     org_rocksdb_RocksDB
+ * Method:    destroyColumnFamilyHandle
+ * Signature: (JJ)V;
+ */
+void Java_org_rocksdb_RocksDB_destroyColumnFamilyHandle(JNIEnv* env,
+                                                        jobject /*jdb*/,
+                                               jlong jdb_handle,
```

Comment:
Nit: indentation.

Reply:
I need two reviewer approve to merge it?